pages/ugm2024.md

```diff
@@ -0,0 +1,77 @@
+* Talks related to developing and using MDAnalysis for diverse scientific applications
+* Tutorials diving into new features
+* Direct access to the [MDAnalysis core developer team][coredev] to help shape future development of MDAnalysis
+* Networking opportunities to foster new collaborations
+* Hackathon and other social outings
+
+## Keynote Speakers
+Keynote speakers will be announced in Spring 2024.
+
+## Program
+The agenda will be announced in Spring 2024.
+
+## Registration and Abstracts
+Registration details and a call for abstracts will be announced shortly.
+
+## Travel and Accommodation
+Participants are expected to make their own travel and accommodation arrangements.
+
+Citizens of several countries may travel to the United Kingdom to attend a meeting without obtaining a visa. Please visit the [gov.uk][https://www.gov.uk/browse/visas-immigration/tourist-short-stay-visas] website for more information about travel visas. Attendees requiring a letter clarifying the purpose of travel should [contact the MDAnalysis project manager][email]. 
+
+## Health and Safety
+We are committed to providing a safe, comfortable, and healthy environment for the MDAnalysis UGM and have looked to local and international entities and other organizations in the open source community to develop onsite safety measures for UGM attendees.
+
+
+### Testing
+A limited number of rapid self-tests will be made available at check-in for those who would like to test themselves for COVID-19 during the meeting.
+
+*Testing Positive for COVID-19*
+
+Individuals who are COVID-positive should not attend the UGM. If you receive a positive COVID-19 test result during the UGM, please [notify the MDAnalysis UGM organizing committee][ugm_email] and do not return to the event while you are still COVID-positive.
+
+### Face Coverings
+Face coverings are not required, but are welcomed. We view face coverings as a personal choice and will support attendees in their choice to wear or not to wear a face covering.
+
+### Social Distancing and Hygiene
+While we will be unable to enforce social distancing guidelines during the UGM, there is extra seating capacity at the venue that allows for voluntary distancing. At check-in, you will have the option to indicate your comfort level with social distancing on your name tag.
+
+## Code of Conduct
+All members of the MDAnalysis community and participants in this UGM are expected to abide by the MDAnalysis [Code of Conduct][coc] (CoC).
+
+If you are attending the UGM and wish to make a report, you may contact the CoC Committee members or other event staff/meetup organizers so that they can take any appropriate immediate response. If you would prefer not to do that, please [submit a report][coc_report] to MDAnalysis. The MDAnalysis CoC Committee includes Jenna Swarthout Goddard, Micaela Matta and Richard Gowers, who will identify themselves at the event.
```

Comment:
You could link the CoC Committee to the Roles page instead of listing everyone here, so that information is in a single place.

However, if it's best practice to write is out explicitly then follow best practice!

Reply:
Good idea!